packages/bacon/package.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: bacon
+description: Bacon is a background rust code checker
+homepage: https://github.com/Canop/bacon
+licenses:
+  - AGPL-3.0-or-later
+languages:
+  - Rust
+categories:
+  - Runtime
```

Comment:
I think `Linter` would be a more appropriate category here? The `Runtime` category is not really used yet by Mason and is more geared towards software that provide a traditional runtime, like Node, Python, Deno, etc.

Reply:
Sounds good, I was unsure about this as well. Let me fix it :)

Reply:
Done in https://github.com/mason-org/mason-registry/pull/5774/commits/8838fbfe4b8406015979bb7c572e96b7dfe98128